test/invariants/handlers/RouterHandler.sol

```diff
@@ -36,6 +38,7 @@ contract RouterHandler is Test, TypedDataSignature {
         owner = msg.sender;
         router = router_;
 
+        (user, userPrivateKey) = makeAddrAndKey('User');
```

Comment:
Is there any reason that this `user` is set to be fixed in the `RouterHandler` constructor, instead of using `actorSeed` to continuously generate a new user for each invariant test?

Reply:
I found the invariant `invariant_matchedAgentsLength` would fail if we use `actorSeed` for delegation-related functions (eg. `executeBySigXXX` & `executeForXXX`). The reason is because delegators do not have an agent. Considering the invariant is still valid in delegation scenarios, the `user` is used in these scenarios instead.

Reply:
Since Solidity does not allow overload for the modifier, I added new modifiers for delegation scenarios.